diff --git a/templates/fortune/showCategory.html.twig b/templates/fortune/showCategory.html.twig
index 20e488b..bb8c498 100644
--- a/templates/fortune/showCategory.html.twig
+++ b/templates/fortune/showCategory.html.twig
@@ -18,7 +18,7 @@
                 </tr>
             </thead>
             <tbody>
-            {% for fortuneCookie in category.fortuneCookies %}
+            {% for fortuneCookie in category.fortuneCookiesStillInProduction %}
                 <tr class="hover:bg-slate-200">
                     <td class="border p-4">
                         {{ fortuneCookie.fortune }}
