diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index 334f752..f20f4f8 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -57,6 +57,7 @@ class CategoryRepository extends ServiceEntityRepository
         return $this->addFortuneCookieJoinAndSelect()
             ->andWhere('category.id = :id')
             ->setParameter('id', $id)
+            ->orderBy('RAND()', Criteria::ASC)
             ->getQuery()
             ->getOneOrNullResult();
     }
