diff --git a/src/Model/CategoryFortuneStats.php b/src/Model/CategoryFortuneStats.php
new file mode 100644
index 0000000..c7c58b9
--- /dev/null
+++ b/src/Model/CategoryFortuneStats.php
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Model;
+
+class CategoryFortuneStats
+{
+    public function __construct(
+        public int $fortunesPrinted,
+        public float $fortunesAverage,
+        public string $categoryName,
+    )
+    {
+    }
+}
