diff --git a/src/Controller/FortuneController.php b/src/Controller/FortuneController.php
index 21c0f5f..eff5405 100644
--- a/src/Controller/FortuneController.php
+++ b/src/Controller/FortuneController.php
@@ -5,6 +5,7 @@ namespace App\Controller;
 use App\Entity\Category;
 use App\Repository\CategoryRepository;
 use App\Repository\FortuneCookieRepository;
+use Doctrine\ORM\EntityManagerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
@@ -13,7 +14,7 @@ use Symfony\Component\Routing\Annotation\Route;
 class FortuneController extends AbstractController
 {
     #[Route('/', name: 'app_homepage')]
-    public function index(Request $request, CategoryRepository $categoryRepository): Response
+    public function index(Request $request, CategoryRepository $categoryRepository, EntityManagerInterface $entityManager): Response
     {
         $searchTerm = $request->query->get('q');
         if ($searchTerm) {
