diff --git a/src/Controller/FortuneController.php b/src/Controller/FortuneController.php
index 8995450..ac96282 100644
--- a/src/Controller/FortuneController.php
+++ b/src/Controller/FortuneController.php
@@ -16,6 +16,8 @@ class FortuneController extends AbstractController
     #[Route('/', name: 'app_homepage')]
     public function index(Request $request, CategoryRepository $categoryRepository, EntityManagerInterface $entityManager): Response
     {
+        $entityManager->getFilters()
+            ->enable('fortuneCookie_discontinued');
         $searchTerm = $request->query->get('q');
         if ($searchTerm) {
             $categories = $categoryRepository->search($searchTerm);
