diff --git a/src/Controller/FortuneController.php b/src/Controller/FortuneController.php
index 855d6eb..0f0ee17 100644
--- a/src/Controller/FortuneController.php
+++ b/src/Controller/FortuneController.php
@@ -18,7 +18,7 @@ class FortuneController extends AbstractController
     {
         $entityManager->getFilters()
             ->enable('fortuneCookie_discontinued')
-            ->setParameter('discontinued', false);
+            ->setParameter('discontinued', true);
         $searchTerm = $request->query->get('q');
         if ($searchTerm) {
             $categories = $categoryRepository->search($searchTerm);
