diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index 2c20104..62c55e3 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -80,6 +80,11 @@ class CategoryRepository extends ServiceEntityRepository
         }
     }
 
+    private function addGroupByCategory(QueryBuilder $qb): QueryBuilder
+    {
+
+    }
+
     private function addFortuneCookieJoinAndSelect(QueryBuilder $qb = null): QueryBuilder
     {
         return ($qb ?? $this->createQueryBuilder('category'))
