diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index 752deaf..204ce57 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -84,6 +84,11 @@ class CategoryRepository extends ServiceEntityRepository
         }
     }
 
+    private function addGroupByCategory(QueryBuilder $qb): QueryBuilder
+    {
+
+    }
+
     private function addFortuneCookieJoinAndSelect(QueryBuilder $qb = null): QueryBuilder
     {
         return ($qb ?? $this->createQueryBuilder('category'))
