diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index a92a9b0..380352d 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -32,6 +32,10 @@ class CategoryRepository extends ServiceEntityRepository
         $qb = $this->createQueryBuilder('category');
 //        static::addFortuneCookieJoinAndSelect($qb);
         static::addOrderByCategoryName($qb);
+        $qb
+            ->addSelect('category.id, category.name, category.iconKey, COUNT(fortuneCookie.id) AS fortuneCookiesTotal')
+            ->leftJoin('category.fortuneCookies', 'fortuneCookie')
+            ->addGroupBy('category.id');
         $query = $qb->getQuery();
 
         return $query->getResult();
