diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index bf4fe10..9f98b81 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -43,9 +43,8 @@ class CategoryRepository extends ServiceEntityRepository
     {
         $termList = explode(' ', $term);
         $qb = $this->addOrderByCategoryName();
-        $qb = $this->addGroupByCategory();
 
-        return $this->addFortuneCookieJoinAndSelect($qb)
+        return $this->addGroupByCategory($qb)
             ->andWhere('category.name LIKE :searchTerm OR category.name IN (:termList) OR category.iconKey LIKE :searchTerm OR fortuneCookie.fortune LIKE :searchTerm')
             ->setParameter('searchTerm', '%'.$term.'%')
             ->setParameter('termList', $termList)
