diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index ed36bc4..719e647 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -35,7 +35,6 @@ class CategoryRepository extends ServiceEntityRepository
             ->leftJoin('category.fortuneCookies', 'fortuneCookie')
             ->addGroupBy('category.id');
         $query = $qb->getQuery();
-        dd($query->getResult());
 
         return $query->getResult();
     }
diff --git a/templates/fortune/homepage.html.twig b/templates/fortune/homepage.html.twig
index 8700a0f..ac35000 100644
--- a/templates/fortune/homepage.html.twig
+++ b/templates/fortune/homepage.html.twig
@@ -5,9 +5,10 @@
 <h1 class="text-center my-8 text-2xl text-slate-800 font-bold">Query for your Fortune</h1>
 <p class="text-center">Review categories for available cookie fortunes</p>
 <div class="xl:container xl:mx-auto grid grid-cols-2 gap-6 px-10 mt-8 mb-10">
-    {% for category in categories %}
+    {% for categoryData in categories %}
+        {% set category = categoryData[0] %}
         <a class="bg-orange-400 hover:bg-orange-500 text-white text-center rounded-full p-4" href="{{ path('app_category_show', {'id':  category.id}) }}">
-            <span class="fa {{ category.iconKey }}"></span> <span class="font-bold text-lg">{{ category.name }}</span>  ({{ category.fortuneCookies|length }})
+            <span class="fa {{ category.iconKey }}"></span> <span class="font-bold text-lg">{{ category.name }}</span>  ({{ categoryData.fortuneCookiesTotal }})
         </a>
     {% else %}
         <h3>Hmm... there don't seem to be any fortunes</h3>
