diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index 62c55e3..b12db25 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -26,11 +26,8 @@ class CategoryRepository extends ServiceEntityRepository
     public function findAllOrdered(): array
     {
 //        $dql = 'SELECT category FROM App\Entity\Category as category ORDER BY category.name DESC';
-        $qb = $this->createQueryBuilder('category')
-            ->addOrderBy('category.name', Criteria::DESC)
-            ->addSelect('COUNT(fortuneCookie.id) AS fortuneCookiesTotal')
-            ->leftJoin('category.fortuneCookies', 'fortuneCookie')
-            ->addGroupBy('category.id');
+        $qb = $this->addGroupByCategory()
+            ->addOrderBy('category.name', Criteria::DESC);
         $query = $qb->getQuery();
 
         return $query->getResult();
@@ -80,9 +77,12 @@ class CategoryRepository extends ServiceEntityRepository
         }
     }
 
-    private function addGroupByCategory(QueryBuilder $qb): QueryBuilder
+    private function addGroupByCategory(QueryBuilder $qb = null): QueryBuilder
     {
-
+        return ($qb ?? $this->createQueryBuilder('category'))
+            ->addSelect('COUNT(fortuneCookie.id) AS fortuneCookiesTotal')
+            ->leftJoin('category.fortuneCookies', 'fortuneCookie')
+            ->addGroupBy('category.id');
     }
 
     private function addFortuneCookieJoinAndSelect(QueryBuilder $qb = null): QueryBuilder
