diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index 1e46053..86d9738 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -30,8 +30,12 @@ class CategoryRepository extends ServiceEntityRepository
     {
 //        $dql = 'SELECT category FROM App\Entity\Category as category ORDER BY category.name DESC';
         $qb = $this->createQueryBuilder('category')
-            ->addOrderBy('category.name', Criteria::DESC);
+            ->addOrderBy('category.name', Criteria::DESC)
+            ->addSelect('COUNT(fortuneCookie.id) AS fortuneCookiesTotal')
+            ->leftJoin('category.fortuneCookies', 'fortuneCookie')
+            ->addGroupBy('category.id');
         $query = $qb->getQuery();
+        dd($query->getResult());
 
         return $query->getResult();
     }
