diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index 2250561..a92a9b0 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -30,7 +30,7 @@ class CategoryRepository extends ServiceEntityRepository
     {
 //        $dql = 'SELECT category FROM App\Entity\Category as category ORDER BY category.name DESC';
         $qb = $this->createQueryBuilder('category');
-        static::addFortuneCookieJoinAndSelect($qb);
+//        static::addFortuneCookieJoinAndSelect($qb);
         static::addOrderByCategoryName($qb);
         $query = $qb->getQuery();
 
