diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index 713aeff..76b28f7 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -43,6 +43,7 @@ class CategoryRepository extends ServiceEntityRepository
     {
         $termList = explode(' ', $term);
         $qb = $this->addOrderByCategoryName();
+        $qb = $this->addGroupByCategory();
 
         return $this->addFortuneCookieJoinAndSelect($qb)
             ->andWhere('category.name LIKE :searchTerm OR category.name IN (:termList) OR category.iconKey LIKE :searchTerm OR fortuneCookie.fortune LIKE :searchTerm')
