diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index c2f819e..4301e70 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -91,7 +91,7 @@ class CategoryRepository extends ServiceEntityRepository
     private static function addGroupByCategory(QueryBuilder $qb): QueryBuilder
     {
         return $qb
-            ->addSelect('category.id, category.name, category.iconKey, COUNT(fortuneCookie.id) AS fortuneCookiesTotal')
+            ->addSelect('category, COUNT(fortuneCookie.id) AS fortuneCookiesTotal')
             ->leftJoin('category.fortuneCookies', 'fortuneCookie')
             ->addGroupBy('category.id');
     }
