diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index 340ccb4..f8c9f65 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -41,6 +41,7 @@ class CategoryRepository extends ServiceEntityRepository
      */
     public function search(string $term): array
     {
+        $termList = explode(' ', $term);
         $qb = $this->addOrderByCategoryName();
 
         return $this->addFortuneCookieJoinAndSelect($qb)
