diff --git a/src/Controller/FortuneController.php b/src/Controller/FortuneController.php
index 760db70..74d678a 100644
--- a/src/Controller/FortuneController.php
+++ b/src/Controller/FortuneController.php
@@ -27,8 +27,10 @@ class FortuneController extends AbstractController
     }
 
     #[Route('/category/{id}', name: 'app_category_show')]
-    public function showCategory(Category $category): Response
+    public function showCategory(int $id, CategoryRepository $categoryRepository): Response
     {
+        $category = $categoryRepository->findWithFortunesJoin($id);
+
         return $this->render('fortune/showCategory.html.twig',[
             'category' => $category
         ]);
