diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index dd16e5a..526657e 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -29,6 +29,7 @@ class CategoryRepository extends ServiceEntityRepository
     {
 //        $dql = 'SELECT category FROM App\Entity\Category as category ORDER BY category.name DESC';
         $qb = $this->createQueryBuilder('category')
+            ->addSelect('fortuneCookie')
             ->leftJoin('category.fortuneCookies', 'fortuneCookie')
             ->addOrderBy('category.name', Criteria::DESC);
         $query = $qb->getQuery();
