diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index 46afb2e..0300d77 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -52,7 +52,12 @@ class CategoryRepository extends ServiceEntityRepository
 
     public function findWithFortunesJoin(int $id): ?Category
     {
-        
+        return $this->createQueryBuilder('category')
+            ->addSelect('fortuneCookie')
+            ->leftJoin('category.fortuneCookies', 'fortuneCookie')
+            ->andWhere('category.id = :id')->setParameter('id', $id)
+            ->getQuery()
+            ->getOneOrNullResult();
     }
 
     public function save(Category $entity, bool $flush = false): void
