diff --git a/src/Controller/FortuneController.php b/src/Controller/FortuneController.php
index 74d678a..2d8162a 100644
--- a/src/Controller/FortuneController.php
+++ b/src/Controller/FortuneController.php
@@ -30,6 +30,9 @@ class FortuneController extends AbstractController
     public function showCategory(int $id, CategoryRepository $categoryRepository): Response
     {
         $category = $categoryRepository->findWithFortunesJoin($id);
+        if (!$category) {
+            throw $this->createNotFoundException('Category not found!');
+        }
 
         return $this->render('fortune/showCategory.html.twig',[
             'category' => $category
