diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index 595044f..46afb2e 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -50,6 +50,11 @@ class CategoryRepository extends ServiceEntityRepository
             ->getResult();
     }
 
+    public function findWithFortunesJoin(int $id): ?Category
+    {
+        
+    }
+
     public function save(Category $entity, bool $flush = false): void
     {
         $this->getEntityManager()->persist($entity);
