diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index f023dd9..3c40c7e 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -31,7 +31,6 @@ class CategoryRepository extends ServiceEntityRepository
         $qb = $this->createQueryBuilder('category')
             ->addOrderBy('category.name', Criteria::DESC);
         $query = $qb->getQuery();
-        dd($query->getDQL());
 
         return $query->getResult();
     }
