diff --git a/src/Repository/FortuneCookieRepository.php b/src/Repository/FortuneCookieRepository.php
index 8fcc52d..720edb3 100644
--- a/src/Repository/FortuneCookieRepository.php
+++ b/src/Repository/FortuneCookieRepository.php
@@ -35,6 +35,7 @@ class FortuneCookieRepository extends ServiceEntityRepository
 //            ->getSingleResult();
 
         $conn = $this->getEntityManager()->getConnection();
+        dd($conn);
 
         return $result;
     }
