diff --git a/src/Repository/FortuneCookieRepository.php b/src/Repository/FortuneCookieRepository.php
index 7f4f4cf..ec6943c 100644
--- a/src/Repository/FortuneCookieRepository.php
+++ b/src/Repository/FortuneCookieRepository.php
@@ -42,6 +42,7 @@ class FortuneCookieRepository extends ServiceEntityRepository
 
         $conn = $this->getEntityManager()->getConnection();
         $sql = 'SELECT * FROM fortune_cookie';
+        $stmt = $conn->prepare($sql);
 
         return $result;
     }
