diff --git a/src/Repository/FortuneCookieRepository.php b/src/Repository/FortuneCookieRepository.php
index ec6943c..2e8c366 100644
--- a/src/Repository/FortuneCookieRepository.php
+++ b/src/Repository/FortuneCookieRepository.php
@@ -43,6 +43,7 @@ class FortuneCookieRepository extends ServiceEntityRepository
         $conn = $this->getEntityManager()->getConnection();
         $sql = 'SELECT * FROM fortune_cookie';
         $stmt = $conn->prepare($sql);
+        $result = $stmt->executeQuery();
 
         return $result;
     }
