diff --git a/src/Repository/FortuneCookieRepository.php b/src/Repository/FortuneCookieRepository.php
index fe06c0c..be0a503 100644
--- a/src/Repository/FortuneCookieRepository.php
+++ b/src/Repository/FortuneCookieRepository.php
@@ -40,6 +40,8 @@ class FortuneCookieRepository extends ServiceEntityRepository
 //            ->getQuery()
 //            ->getSingleResult();
 
+        $conn = $this->getEntityManager()->getConnection();
+
         return $result;
     }
 
