diff --git a/src/Repository/FortuneCookieRepository.php b/src/Repository/FortuneCookieRepository.php
index 7a62409..be0a503 100644
--- a/src/Repository/FortuneCookieRepository.php
+++ b/src/Repository/FortuneCookieRepository.php
@@ -41,7 +41,6 @@ class FortuneCookieRepository extends ServiceEntityRepository
 //            ->getSingleResult();
 
         $conn = $this->getEntityManager()->getConnection();
-        dd($conn);
 
         return $result;
     }
