diff --git a/src/Repository/FortuneCookieRepository.php b/src/Repository/FortuneCookieRepository.php
index 697f494..fe06c0c 100644
--- a/src/Repository/FortuneCookieRepository.php
+++ b/src/Repository/FortuneCookieRepository.php
@@ -25,20 +25,20 @@ class FortuneCookieRepository extends ServiceEntityRepository
 
     public function countNumberPrintedForCategory(Category $category): CategoryFortuneStats
     {
-        $result = $this->createQueryBuilder('fortuneCookie')
-            ->select(sprintf(
-                'NEW %s(
-                    SUM(fortuneCookie.numberPrinted),
-                    AVG(fortuneCookie.numberPrinted),
-                    category.name
-                )',
-                CategoryFortuneStats::class
-            ))
-            ->innerJoin('fortuneCookie.category', 'category')
-            ->andWhere('fortuneCookie.category = :category')
-            ->setParameter('category', $category)
-            ->getQuery()
-            ->getSingleResult();
+//        $result = $this->createQueryBuilder('fortuneCookie')
+//            ->select(sprintf(
+//                'NEW %s(
+//                    SUM(fortuneCookie.numberPrinted),
+//                    AVG(fortuneCookie.numberPrinted),
+//                    category.name
+//                )',
+//                CategoryFortuneStats::class
+//            ))
+//            ->innerJoin('fortuneCookie.category', 'category')
+//            ->andWhere('fortuneCookie.category = :category')
+//            ->setParameter('category', $category)
+//            ->getQuery()
+//            ->getSingleResult();
 
         return $result;
     }
