diff --git a/src/Repository/FortuneCookieRepository.php b/src/Repository/FortuneCookieRepository.php
index be0a503..7f4f4cf 100644
--- a/src/Repository/FortuneCookieRepository.php
+++ b/src/Repository/FortuneCookieRepository.php
@@ -41,6 +41,7 @@ class FortuneCookieRepository extends ServiceEntityRepository
 //            ->getSingleResult();
 
         $conn = $this->getEntityManager()->getConnection();
+        $sql = 'SELECT * FROM fortune_cookie';
 
         return $result;
     }
