diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index b5c90a0..9d2353f 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -42,9 +42,10 @@ class CategoryRepository extends ServiceEntityRepository
      */
     public function search(string $term): array
     {
-        return $this->createQueryBuilder('category')
-            ->addSelect('fortuneCookie')
-            ->leftJoin('category.fortuneCookies', 'fortuneCookie')
+        $qb = $this->createQueryBuilder('category');
+        static::addFortuneCookieJoinAndSelect($qb);
+
+        return $qb
             ->andWhere('category.name LIKE :searchTerm OR category.iconKey LIKE :searchTerm OR fortuneCookie.fortune LIKE :searchTerm')
             ->setParameter('searchTerm', '%'.$term.'%')
             ->addOrderBy('category.name', Criteria::DESC)
