diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index 0578c60..e3bec58 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -81,7 +81,9 @@ class CategoryRepository extends ServiceEntityRepository
 
     private function addFortuneCookieJoinAndSelect(QueryBuilder $qb): QueryBuilder
     {
-
+        return $qb
+            ->addSelect('fortuneCookie')
+            ->leftJoin('category.fortuneCookies', 'fortuneCookie');
     }
 
 //    /**
