diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index 55a29f1..1898afb 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -82,6 +82,11 @@ class CategoryRepository extends ServiceEntityRepository
             ->leftJoin('category.fortuneCookies', 'fortuneCookie');
     }
 
+    private function addOrderByCategoryName(QueryBuilder $qb = null): QueryBuilder
+    {
+
+    }
+
 //    /**
 //     * @return Category[] Returns an array of Category objects
 //     */
