diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index cffed14..99879d9 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -82,7 +82,9 @@ class CategoryRepository extends ServiceEntityRepository
 
     private function addFortuneCookieJoinAndSelect(QueryBuilder $qb): QueryBuilder
     {
-
+        return $qb
+            ->addSelect('fortuneCookie')
+            ->leftJoin('category.fortuneCookies', 'fortuneCookie');
     }
 
 //    /**
