diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index 0d43be8..49e7c20 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -53,9 +53,9 @@ class CategoryRepository extends ServiceEntityRepository
 
     public function findWithFortunesJoin(int $id): ?Category
     {
-        return $this->createQueryBuilder('category')
-            ->addSelect('fortuneCookie')
-            ->leftJoin('category.fortuneCookies', 'fortuneCookie')
+        $qb = $this->createQueryBuilder('category');
+
+        return $this->addFortuneCookieJoinAndSelect($qb)
             ->andWhere('category.id = :id')->setParameter('id', $id)
             ->getQuery()
             ->getOneOrNullResult();
