diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index 77dfb9c..f1671b9 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -91,7 +91,7 @@ class CategoryRepository extends ServiceEntityRepository
 
     private static function addOrderByCategoryName(QueryBuilder $qb): QueryBuilder
     {
-
+        return $qb->addOrderBy('category.name', Criteria::DESC);
     }
 
 //    /**
