diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index 49e7c20..55a29f1 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -41,9 +41,7 @@ class CategoryRepository extends ServiceEntityRepository
      */
     public function search(string $term): array
     {
-        $qb = $this->createQueryBuilder('category');
-
-        return $this->addFortuneCookieJoinAndSelect($qb)
+        return $this->addFortuneCookieJoinAndSelect()
             ->andWhere('category.name LIKE :searchTerm OR category.iconKey LIKE :searchTerm OR fortuneCookie.fortune LIKE :searchTerm')
             ->setParameter('searchTerm', '%'.$term.'%')
             ->addOrderBy('category.name', Criteria::DESC)
@@ -53,9 +51,7 @@ class CategoryRepository extends ServiceEntityRepository
 
     public function findWithFortunesJoin(int $id): ?Category
     {
-        $qb = $this->createQueryBuilder('category');
-
-        return $this->addFortuneCookieJoinAndSelect($qb)
+        return $this->addFortuneCookieJoinAndSelect()
             ->andWhere('category.id = :id')->setParameter('id', $id)
             ->getQuery()
             ->getOneOrNullResult();
@@ -79,9 +75,9 @@ class CategoryRepository extends ServiceEntityRepository
         }
     }
 
-    private function addFortuneCookieJoinAndSelect(QueryBuilder $qb): QueryBuilder
+    private function addFortuneCookieJoinAndSelect(QueryBuilder $qb = null): QueryBuilder
     {
-        return $qb
+        return ($qb ?? $this->createQueryBuilder('category'))
             ->addSelect('fortuneCookie')
             ->leftJoin('category.fortuneCookies', 'fortuneCookie');
     }
