diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index 2037cc3..c4c74d1 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -83,6 +83,11 @@ class CategoryRepository extends ServiceEntityRepository
             ->leftJoin('category.fortuneCookies', 'fortuneCookie');
     }
 
+    private function addOrderByCategoryName(QueryBuilder $qb = null): QueryBuilder
+    {
+
+    }
+
 //    /**
 //     * @return Category[] Returns an array of Category objects
 //     */
