diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index aaea5aa..cffed14 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -5,6 +5,7 @@ namespace App\Repository;
 use App\Entity\Category;
 use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
 use Doctrine\Common\Collections\Criteria;
+use Doctrine\ORM\QueryBuilder;
 use Doctrine\Persistence\ManagerRegistry;
 
 /**
@@ -79,6 +80,11 @@ class CategoryRepository extends ServiceEntityRepository
         }
     }
 
+    private function addFortuneCookieJoinAndSelect(QueryBuilder $qb): QueryBuilder
+    {
+
+    }
+
 //    /**
 //     * @return Category[] Returns an array of Category objects
 //     */
