diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index c4c74d1..2ffe8c0 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -85,7 +85,8 @@ class CategoryRepository extends ServiceEntityRepository
 
     private function addOrderByCategoryName(QueryBuilder $qb = null): QueryBuilder
     {
-
+        return ($qb ?? $this->createQueryBuilder('category'))
+            ->addOrderBy('category.name', Criteria::DESC);
     }
 
 //    /**
