diff --git a/src/Repository/CategoryRepository.php b/src/Repository/CategoryRepository.php
index 7d0cf7b..0547a20 100644
--- a/src/Repository/CategoryRepository.php
+++ b/src/Repository/CategoryRepository.php
@@ -41,7 +41,7 @@ class CategoryRepository extends ServiceEntityRepository
     public function search(string $term): array
     {
         return $this->createQueryBuilder('category')
-            ->andWhere('category.name LIKE :searchTerm')
+            ->andWhere('category.name LIKE :searchTerm OR category.iconKey LIKE :searchTerm')
             ->setParameter('searchTerm', '%'.$term.'%')
             ->getQuery()
             ->getResult();
